weave/trace/errors.py

```diff
@@ -23,3 +23,6 @@ class WeaveDefinitionError(Error): ...
 
 
 class WeaveWandbAuthenticationException(Error): ...
+
+
+class WeaveConfigurationError(Error): ...
```

Comment:
legacy

Reply:
yeah, it's moved in the next pr:
https://github.com/wandb/weave/pull/2471